node/config/types.go

```diff
@@ -115,6 +115,8 @@ type DealmakingConfig struct {
 	// This includes the time the deal will need to get transferred and published
 	// before being assigned to a sector
 	ExpectedSealDuration Duration
+	// Whether new sectors are sealed to pack incoming deals
```

Comment:
```suggestion
	// Whether new sectors are created to pack incoming deals
```

Reply:
Possibly worth elaborating what happens when this is set to false, and why one would do that